openerp/addons/base/ir/ir_model.py

```diff
@@ -237,6 +237,45 @@ class CustomModel(models.Model):
 
         CustomModel._build_model(self.pool, cr)
 
+    def load(self, cr, uid, fields, data, context=None):
+        """
+        Attempts to load the data matrix, and returns a list of ids (or
+        ``False`` if there was an error and no id could be generated) and a
+        list of messages.
+
+        The ids are those of the records created and saved (in database), in
+        the same order they were extracted from the file. They can be passed
+        directly to :meth:`~read`
+
+        :param fields: list of fields to import, at the same index as the corresponding data
+        :type fields: list(str)
+        :param data: row-major matrix of data to import
+        :type data: list(list(str))
+        :param dict context:
+        :returns: {ids: list(int)|False, messages: [Message]}
```

Comment:
Modify the documentation:

    Load the data matrix, and return a list of ids and a list of messages.

    Note that the processing of each line commits its result to the database, because this is what this model does.
    Therefore, one cannot rollback changes if an error occurs.